README.md

```diff
@@ -1 +1,58 @@
-# pypowsybl-grid2opbackend
\ No newline at end of file
+# pypowsybl-grid2opbackend integration (AIRGo project)
```

Comment:
would be nice to have something showing how to use the backend with the provided environment (which command to type in python)